site/src/components/form.md

```diff
@@ -43,58 +43,233 @@ It can be used to hold `.field-body` containers, which holds the individual elem
 </fieldset>
 ```
 
-## Form Validation
+## Required Indicator
 
-`.field-error` is placed after the element for form validation error messages.
+For elements that are required for form submission, you can add a span element with the `.required-indicator` class inside the label or adjacent to the element.
 
 ```html
 <form>
-	<p class="\visually-hidden">Required fields are marked with asterisk/star</p>
+	<p class="visually-hidden">Required fields are marked with asterisk/star</p>
+	<fieldset class="field">
+		<legend class="field-label">Radio group <span class="required-indicator"></span></legend>
+		<div class="field-body">
+			<label class="radio">
+				<input checked name="question-1" type="radio" class="radio-dot" value="Yes" required />
+				<span class="radio-label-text">Yes</span>
+			</label>
+			<label class="radio">
+				<input checked name="question-1" type="radio" class="radio-dot" value="No" />
+				<span class="radio-label-text">No</span>
+			</label>
+		</div>
+	</fieldset>
+</form>
+```
+
+## Form Behavior
+
+Use the `form-behavior` component within a form to ensure accessible form validation. Form elements should be contained by `.field` and `.field-body` class.
+When the form do not require any edits (i.e the only action is to submit), you can add a `new` attribute to override the validation.
```

Comment:
Quick grammar feedback: should this be `When the form does not require`?

Reply:
Updated